src/pykeen/models/unimodal/boxe.py

```diff
@@ -0,0 +1,132 @@
+# -*- coding: utf-8 -*-
+
+"""Implementation of the BoxE model."""
+
+from typing import Any, ClassVar, Mapping, Optional
+
+from torch.nn.init import uniform_
+
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...losses import NSSALoss
+from ...models import ERModel
+from ...nn.emb import EmbeddingSpecification
+from ...nn.init import uniform_norm_
+from ...nn.modules import BoxEInteraction
+from ...typing import Hint, Initializer
+
+__all__ = [
+    "BoxE",
+]
+
+
+class BoxE(ERModel):
+    r"""An implementation of BoxE from [abboud2020]_.
+
+    ---
+    citation:
+        author: Abboud
+        year: 2020
+        link: https://arxiv.org/abs/2007.06267
+        github: ralphabb/BoxE
+    """
+
+    #: The default strategy for optimizing the model's hyper-parameters
+    hpo_default: ClassVar[Mapping[str, Any]] = dict(
+        embedding_dim=DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE,
+        p=dict(type=int, low=1, high=2),
+    )
+
+    loss_default = NSSALoss
+    loss_default_kwargs = dict(margin=3, adversarial_temperature=2.0, reduction="sum")
+
+    def __init__(
+        self,
+        *,
+        embedding_dim: int = 200,
```

Comment:
Is there any specific reason why this is 200? If not, I would prefer a embedding dimension which allows utilization of tensor cores (multiple of 8, larger powers of two are even better for performance)

```suggestion
        embedding_dim: int = 256,
```

Reply:
The main reason the default is 200 is because this is the standard value in the literature on e.g., YAGO3-10 and higher-arity benchmarks. No problem with changing it.

Reply:
Your argument is also quite good :confused: 

I think, I would still change it, since we have the experiment configurations (the YAMLs mentioned below) which explicitly provide all settings to re-run a certain experiment.